fair/concentration_driven.py

```diff
@@ -26,25 +26,15 @@ def run(inp_df, cfg):
 
     res_dict = _run_numpy(*arg_list)
 
-    inp_df = inp_df.iloc[:, inp_df.columns.astype("str").str.lower().argsort()]
-    inp_df = inp_df.iloc[inp_df.index.astype("str").str.lower().argsort()]
-
-    emissions_df, RF_df, T_df, alpha_df = unifiedtools.create_output_dataframes(
-        inp_df,
-        res_dict["emissions"],
-        res_dict["RF"],
-        res_dict["T"],
-        res_dict["alpha"],
-        arg_list[-2],
-    )
-    res_df_dict = {
-        "emissions": emissions_df,
-        "C": inp_df,
-        "RF": RF_df,
-        "T": T_df,
-        "alpha": alpha_df,
-    }
-    return res_df_dict
+    res_df_iamc_compliant = unifiedtools.create_output_dataframe_aimc_compliant(    inp_df,\
```

Comment:
can you format the code before the next review please, just makes it way easier to read

```suggestion
    res_df_iamc_compliant = unifiedtools.create_output_dataframe_iamc_compliant(    inp_df,\
```